src/query/scan_json_table.cpp

```diff
@@ -294,12 +292,18 @@ namespace cubscan
 	  ASSERT_ERROR ();
 	  return error_code;
 	}
-      if (value_p == NULL || db_value_is_null (value_p))
+      if (value_p == NULL)
 	{
 	  assert (false);
 	  return ER_FAILED;
 	}
 
+      if (db_value_is_null (value_p))
+	{
+	  // input is empty
+	  return NO_ERROR;
```

Comment:
This works only if m_scan_cursor[0].m_is_node_consumed is true. Please add a safe-guard:
```suggestion
          assert (m_scan_cursor[0].m_is_node_consumed);
	  return NO_ERROR;
```

Reply:
yeah. It would be good :)